cli/src/pcluster/templates/awsbatch_builder.py

```diff
@@ -156,7 +157,10 @@ def _launch_template(self, http_tokens):
             self.stack_scope,
             "PclusterComputeEnvironmentLaunchTemplate",
             launch_template_data=ec2.CfnLaunchTemplate.LaunchTemplateDataProperty(
-                metadata_options=ec2.CfnLaunchTemplate.MetadataOptionsProperty(http_tokens=http_tokens)
+                metadata_options=ec2.CfnLaunchTemplate.MetadataOptionsProperty(http_tokens=http_tokens),
+                user_data=Fn.base64(get_user_data_content("../resources/login_node/user_data.sh"))
```

Comment:
Why login nodes user data? Shouldn't it be batch compute env user data?

Reply:
Thanks for the Catch